co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstance.java

```diff
@@ -167,27 +234,40 @@ public void bind() throws IOException {
      * @throws IOException If there is an issue with the underlying socket object or
      *                     methods
      */
-    public void sendMsgs(byte[] data) throws IOException {
-        if (rxMsgsSocket == null) {
-            throw new IllegalStateException("Attempted to send data before opening socket");
-        }
-        DatagramPacket packet = new DatagramPacket(data, data.length, targetAddress, rxMessagePort);
-        rxMsgsSocket.send(packet);
-
+    public void sendV2xMsg(byte[] data) throws IOException {
+        sendPacket(data, rxMessagePort);
     }
 
     /**
-     * Sends time sync data to the Infrastructure Device communications interface configured at construction time.
+     * Sends time sync data to the Infrastrucutre Instance Time Sync port.
+     * 
+     * @param data The binary data to transmit
+     * @throws IOException If there is an issue with the underlying socket object or methods
+     */
+    public void sendTimeSyncMsg(byte[] data) throws IOException {
```

Comment:
Have we defined a class for the Time Sync message? I feel like it might be cleaner to have this (and the interaction one below) take in some type more meaningful than a byte[]. I think it make sense in the v2x message case because byte[] is actually the real type of a V2XMessage, but the time data and (I imagine) the interaction data are more higher-level than that.